content/_index.md

```diff
@@ -27,31 +28,18 @@ images:
 		<img src="https://img.shields.io/github/license/odin-lang/odin.svg">
 	</a>
 </p>
-</center>
+</section>
 
-# The Odin Programming Language
+# Design Goals
 
-
-The Odin programming language is fast, concise, readable, pragmatic and open sourced. It is designed with the intent of creating an alternative to C with the following goals:
+The Odin programming language is designed with the intent of creating an alternative to C with the following goals:
 
 * simplicity
 * high performance
 * built for modern systems
 * joy of programming
 
-<br>
-
-<table>
-<tbody>
-<tr><td>GitHub:</td> <td><a href="https://github.com/odin-lang/Odin">https://github.com/odin-lang/Odin</a></td></tr>
```

Comment:
This was added mainly because they were useful links people for people to see and what many people wanted straight away. Not sure if this is the best presentation however.

Reply:
I added all three links as buttons in the footer. They are also linked elsewhere in the website.